docs/writing-tests/testharness-api.md

```diff
@@ -248,6 +248,18 @@ promise_test(function(t) {
 }, "Another example");
 ```
 
+## Asynchronous Promise Tests ##
+
+`promise_test` are running sequentially. This might causes timeout when too many
+have to run. `async_promise_test` is a variant, running in parallel.
+
+|              | in sequence | in parallel       |
```

Comment:
Thumbs up to having a table explaining how the different test types relate to each other, but in https://github.com/web-platform-tests/rfcs/pull/75#issuecomment-821263444 I have some thoughts about the specific naming. I liked your initial suggestion better :)

Reply:
ACK.

I don't have a strong opinion. I liked my initial proposal "promise_test_parallel", and I also like "async_promise_test" proposed by @annevk, mostly because this make this table looking nice:
Start from `test()`. Want promise? Add `promise_`. Want parallelisme? Add `async_`. Want both? Add both.

I will use the one you two like the most.